README.md

```diff
@@ -6,7 +6,7 @@ MongoDB custom metrics plugin for mackerel.io agent.
 ## Synopsis
 
 ```shell
-mackerel-plugin-mongodb [-host=<host>] [-port=<port>] [-username=<username>] [-password=<password>] [-tempfile=<tempfile>] [-source=<authenticationDatabase>]
+mackerel-plugin-mongodb [-host=<host>] [-port=<port>] [-username=<username>] [-password=<password>] [-tempfile=<tempfile>] [-source=<authenticationDatabase>] [-url=<mongodb://.....>]
```

Comment:
-url option is mutually exclusive with other options. How about splitting the synopsis into two parts?

````
```shell
mackerel-plugin-mongodb [-host=<host>] [-port=<port>] [-username=<username>] [-password=<password>] [-tempfile=<tempfile>] [-source=<authenticationDatabase>]
```

```shell
mackerel-plugin-mongodb [-url=<mongodb://.....>] [-tempfile=<tempfile>]
```
````